pkg/keystore/file/key.go

```diff
@@ -93,6 +93,10 @@ func decryptKey(data []byte, password string) (*ecdsa.PrivateKey, error) {
 	return crypto.DecodeSecp256k1PrivateKey(d)
 }
 
+func DecryptKey(data []byte, password string) (*ecdsa.PrivateKey, error) {
+	return decryptKey(data, password)
+}
+
```

Comment:
you don't need to expose `decryptKey` just for testing purpose and instead just rename `file_test` to `file`.